protoc/CppFileGenerator.cpp

```diff
@@ -220,6 +220,10 @@ void FileGenerator::GenerateBuildDescriptors(Printer* printer) {
     // protobuf_AssignDescriptorsOnce():  The first time it is called, calls
     // AssignDescriptors().  All later times, waits for the first call to
     // complete and then returns.
+
+    // We need to generate different code, depending on the version
+    // of protobuf we compile against
+#if GOOGLE_PROTOBUF_VERSION < 3008000
```

Comment:
Should this not be 3007000? One of the changes appeared here which is in all 3.7.0 variants as I mentioned in my comment:
https://github.com/protocolbuffers/protobuf/commit/8c9e0ff0f711616afa0c1a3252be587a3c0e53c7

Likewise the version matches:
https://github.com/protocolbuffers/protobuf/blob/v3.7.0rc1/src/google/protobuf/stubs/common.h#L84

Reply:
Yep, agreed. Since I don't have every version of protobuf here to test, let's change this to 3007000 :+1: